### PR TITLE
More wizard stuff

### PR DESCRIPTION
**Pull request by Alberyk**
-removes the jaunt spell from the wizard corrupt form
-increases the price of raise dead to two points
-adds a new wizards spell; flesh to stone, that traps your target in a statue for some time
-added a chance of creating a central command report everytime the lich is send back to his heart, telling the possible location of the heart

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/statue.dm [hunk 1]
```diff
@@ -43,9 +49,11 @@
 			desc = "If it takes forever, I will wait for you..."
 
 		var/mob/temporarymob = new (src)
+		temporarymob.name = "imprisioned [L.name]"
```
Comment (Lohikar) [inline code/game/objects/structures/crates_lockers/closets/statue.dm]: Should be imprisoned.

File: code/game/objects/structures/crates_lockers/closets/statue.dm [hunk 2]
```diff
@@ -67,6 +75,11 @@
 		O.forceMove(src.loc)
 
 	for(var/mob/living/M in src)
+		if(imprisoned)
+			if(imprisoned.key)
+				imprisoned.key = M.key
+				qdel(imprisoned)
```
Comment (Lohikar) [inline code/game/objects/structures/crates_lockers/closets/statue.dm]: This should probably be `QDEL_NULL`.

**Comment by Alberyk** [comment]
Changes made.

**Comment by Lohikar** [comment]
One last thing: It's not a blocker, but it would be nice to give the victim some feedback as to what's happening to them, and maybe a rough indication of when the statue is about to release them.

**Comment by Kaedwuff** [comment]
It might also be a good idea to have the game send them a DN in the realm of _'You can't speak while you are a statue!'_ when they try to talk and '_You can't move while you're a statue!_" when they try to emote.  It's just a small bit of feedback to prevent potential confusion that could be caused by the command simply not functioning.

**Comment by Kaedwuff** [comment]
Also, the spell does not function properly on a number of simple mobs, most of them hostile mobs, despite them being valid targets.  This was tested and revealed to be nonfunctional on:
-Slimes
-Spiders
-Space Sharks
-Any simple bots

**Comment by Alberyk** [comment]
fleshtostonetest

The spells seems to work fine in all kind of living mobs. But, I will add the feedback when trying to speak or move.

**Pull request merged**